<launch>

    <!-- Arguments-->
    <arg name="config_path" value="$(find for_franka_ros)/config/topics.yaml"/>


    <!-- Start Gazebo + MoveIt!-->
    <include file="$(find panda_moveit_config)/launch/demo_gazebo.launch">
        <!-- By changing transmission parameter we change controller type-->
        <!-- position, velocity, effort...-->
        <arg name="transmission" default="position"/>
    </include>

    <!-- TODO: Fix this-->
    <!--node pkg="for_franka_ros" type="send_to_kalip" name="send_to_kalip_node" output="screen"/!-->

    <!-- Start arm interface -->
    <node pkg="for_franka_ros"  type="control_arm_servo_node" name="control_arm_servo_node" output="screen">
        <param name="config_path" value="$(arg config_path)" />
        <!-- ROS params for servo ctl-->
        <rosparam command="load" file="$(find for_franka_ros)/config/franka_servo_config.yaml" />
        <rosparam command="load" file="$(find for_franka_ros)/config/pose_tracking.yaml" />
    </node>

    <!-- Start joy node -->
    <node name="joy" pkg="joy" type="joy_node" output="screen"/>
    <node name="arm_joy" pkg="for_franka_ros" type="arm_joy.py" output="screen"/>



</launch>